### PR TITLE
Ben branch

### PR DESCRIPTION
**Pull request by ben-hill33**

### CODE REVIEW

File: automation.py [hunk 1]
```diff
@@ -22,13 +22,14 @@
 python_path = sys.executable
 script_path = os.path.realpath(__file__)
```
Comment (ashcaz) [inline automation.py]: can we delete line 22 and 23 we dont need them anymore

File: README.md [hunk 2]
```diff
@@ -23,13 +36,9 @@ For Mac ver. 10.15.6 or greater:
 
 7. Drag `cron` from finder to the list of apps in Full Disk Access. Ensure `cron` is checked prior to closing the window. 
 
-### Quick Start
-1. On your Terminal, navigate to the smart-files repo
-2. Run `poetry install`
-3. Run `poetry run smart-files` to display options
 
 ### Add Scheduled Sorting Job
```
Comment (ashcaz) [inline README.md:44]: Where did the quick start section go? I dont see anything in here saying "in order to use smart-files dowlaod it using pip/pip3" with a code box showing an example. Can we make that happen?

**Review by ashcaz — CHANGES_REQUESTED** [review]
Left some comment for changes

File: automation.py (hunk 1 above)
Comment (ben-hill33) [reply automation.py]: Just pushed those changes up.

**Review by ashcaz — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (ben-hill33) [reply README.md:44]: Just updated it in the IMPORTANT block

**Review by kmangub — APPROVED** [review]

**Pull request merged**